README.md

```diff
@@ -1,4 +1,5 @@
-# fortepyan :musical_keyboard:
+# Fortepyan :musical_keyboard:
+![GitHub CI](https://github.com/Nospoko/Fortepyan/actions/workflows/ci_tests.yaml/badge.svg?branch=master) [![PyPI version](https://img.shields.io/pypi/v/fortepyan.svg)](https://pypi.org/project/fortepyan/) [![PyPI download month](https://img.shields.io/pypi/dm/fortepyan.svg)](https://pypi.org/project/fortepyan/)
```

Comment:
@roszcz Let me know If you'd like to see some other badges as well!
![image](https://github.com/Nospoko/fortepyan/assets/35274379/761f8ed8-fdca-436f-a00c-2cb570229aff)